amundsen_application/static/js/components/TableDetail/ExploreButton/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
```

Comment:
I connected some of the components and not others. This decision seems somewhat arbitrary. `DataPreviewButton` makes sense to connect since it needs to fetch data, but others like `LineageLink` and `ExploreButton` could go either way. On one hand you need to write more boilerplate code, on the other hand you might need pass in a bunch of props (or pass in the entire table metadata object as a prop).

Any thoughts on which way we should go?

Reply:
I prefer passing the props to child components. 

One way to think of it is that `DataPreviewButton` is calling another endpoint and nothing else in `TableDetail` needs to know about the results of that request, so it makes sense for that to be a connected component. 

However the others will be using data that `TableDetail` will already obtain, and are currently only used on `TableDetail`, so I think it becomes preferable to pass them down.

Reply:
Disconnected all subcomponents except for `DataPreviewButton`